### PR TITLE
Allow all debug options to be flagged on / off as overrides

### PR DESCRIPTION
**Pull request by jordangarcia**
#178 @iansinnott

### CODE REVIEW

**Comment by iansinnott** [comment]
Cool! :+1: Thanks @jordangarcia, looks awesome

File: src/logging.js [hunk 1]
```diff
@@ -11,24 +20,31 @@ exports.dispatchStart = function(type, payload) {
   }
 }
 
-exports.dispatchError = function(error) {
+exports.dispatchError = function(reactorState, error) {
+  console.log('dispatchError, shoud do', getOption(reactorState, 'logDispatches'))
```
Comment (bhamodi) [inline src/logging.js]: "shoud do"?

File: src/logging.js [hunk 2]
```diff
@@ -11,24 +20,31 @@ exports.dispatchStart = function(type, payload) {
...
+  if (!getOption(reactorState, 'logDispatches')) {
+    return
   }
-}
 
-exports.dispatchEnd = function(state) {
   if (console.group) {
-    console.debug('Dispatch done, new state: ', state.toJS())
+    if (getOption(reactorState, 'logDirtyStores')) {
+      console.log('Stores updated:', dirtyStores.toList().toJS())
```
Comment (bhamodi) [inline src/logging.js]: Probably want a space after the colon here.

**Comment by bhamodi** [comment]
Minor nits, but LGTM! :+1:

**Comment by jordangarcia** [comment]
I updated this to always be true for debug and false for non debug.

Can you take a final look @bhamodi

**Comment by bhamodi** [comment]
Yup, changes look good. Just the minor nits left to address @jordangarcia.

**Pull request merged**